proposals/0066-feature-gate-threshold-automation.md

```diff
@@ -0,0 +1,245 @@
+---
+simd: '0066'
+title: Feature Gate Threshold Automation
+authors:
+  - Tyera Eulburg
+  - Joe Caulfield
+category: Standard
+type: Core
+status: Draft
+created: 2023-10-11
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+This SIMD outlines a proposal for automating the feature activation process
+based on a stake-weighted support threshold, rather than manual human action.
+
+## Motivation
+
+Feature gates wrap new cluster functionality, and typically change the rules of
+consensus. As such, a feature gate needs to be supported by a strong majority
+of cluster stake when it is activated, or else it risks partitioning the
+network. The current feature-gate system comprises two steps:
+
+1. An individual key-holder queues a feature gate for activation
+2. The runtime automatically activates the feature on the next epoch boundary
+
+The key-holder is the one who (with the help of the solana-cli) assesses the
+amount of stake that recognizes the feature and decides whether it is safe to
+activate. This is obviously brittle and subject to human error.
+
+This SIMD proposes that the runtime itself replaces the human to assess the
+amount of stake that supports a particular queued feature and only
+automatically activates it when a preset threshold is met. 
+
+## New Terminology
+
+- Feature Gate program: The BPF program that will own all feature accounts and
+possess the capability to revoke them.
```

Comment:
"and possess the capability to revoke them." - I'd remove this part, as it's unnecessary to describe all functionality that the feature gate program provide in its definition here